recipes/filepattern/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "filepattern" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }} 
+
+source:
+  url: https://github.com/PolusAI/filepattern/archive/v{{ version }}.tar.gz
+  sha256: df30c7a6f45a103ab1bcb6947cad5b329af4929080beffc6a590cc6419095834
+
+build:
+  skip: true  # [py2k]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
+
+outputs:
+  - name: filepattern
+    script: build.sh  # [unix]
+    script: bld.bat  # [win]
+    version: {{ version }}
+    requirements:
+      build:
+        - cmake
+        - make  # [not win]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - git
+      host:
+        - python
+        - setuptools
+        - pip
+        - pybind11
+        - pydantic
+      run:
+        - python
+        - pydantic
+
+    test:
+      imports:
+        - filepattern
+      commands:
+        - pip check
+      requires:
+        - pip
+        - pytest
+
+  - name: libfilepattern
+    script: build-lib.sh  # [unix]
+    script: bld-lib.bat  # [win]
+    version: {{ version }}
+    requirements:
+      build:
+        - cmake
+        - make  # [not win]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - git
+    test:
+      commands:
+        - test -f $PREFIX/include/filepattern.h  # [unix]
```

Comment:
I guess it is still slightly off : `- test -f $PREFIX/include/filepattern/filepattern.h # [unix]`